src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -272,7 +273,8 @@
   "Ok": "Ok",
   "OnApplicationUpdate": "On Application Update",
   "OnApplicationUpdateHelpText": "On Application Update",
-  "OnGrab": "On Grab",
+  "OnGrab": "On Release Grab",
+  "OnGrabHelpText": "On Release Grab",
```

Comment:
```suggestion
  "OnGrabHelpText": "On Release Grab by another application via Prowlarr's API",
```

Reply:
Keeping this one as it aligns with the others